steps/compliance/dwp-issuing-office/DWPIssuingOffice.js

```diff
@@ -73,6 +92,25 @@ class DWPIssuingOffice extends SaveToDraftStore {
         'Inverness DRT',
         'Recovery from Estates'
       ]);
+    } else if (getBenefitCode(this.journey.req.session.BenefitType.benefitType) === 'UC') {
```

Comment:
This doesn't look correct to me. Should it only show these options if 'allowRFE' is set to true? What happens if it is false?